content/100-getting-started/01-quickstart.mdx

```diff
@@ -102,10 +102,12 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
 
 1. It creates a new SQL migration file for this migration in the `prisma/migrations` directory.
-1. It runs the SQL migration file against the database.
+2. It runs the SQL migration file against the database.
+3. Runs `prisma generate` under the hood, which installs Prisma Client and generates a Client based on your models.
```

Comment:
I actually think this is a useful change but I realise there's an issue with the way this is currently phrased. the initial sentence says "_did_ three things" (past tense) but then the bullets are in present tense. Personally, I'd be happy to merged this if we change the bullets all to past tense 🙏

Reply:
Hello @nikolasburk I appreciate this suggestion and have made the recommended change.